### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by thakurajayL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,165 +4,147 @@ SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org
...
 
-# Pending Feature List
+# Pending Features
+
+   Data Testing Features
+
+    - Provision data interface to gNBSim Container/POD/executable for data traffic testing
+    - Triggering downlink data from gNB Sim (CI/CD feature as well)
+
+    3gpp features for gNodeB Simulator
```
Comment (gab-arrobo) [inline README.md:103]: I think this should not have indentation (Take a look at the rendered markdown output)
```suggestion
   3gpp features for gNodeB Simulator
```

**Review by gab-arrobo — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+-->
+# Introduction
+
+This repository is part of the SD-Core project. It provides a tool to simulate
+gNodeB and UE by generating NAS and NGAP messages for the configured UEs and 
+call flows.
+
+# Table Of Contents
+  * [Configuration](./gnbsim_config.md)
```
[the remark this answers is not in the document]
Comment (thakurajayL) [reply docs/README.md:13]: We shall be adding more eventually.

File: docs/gnbsim_config.md
[the remark this answers is not in the document]
Comment (thakurajayL) [reply docs/gnbsim_config.md:1]: Agree. Will do the change.

File: README.md [hunk 3]
```diff
@@ -4,165 +4,147 @@ SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org
...
-    - Support to handle Paging Request
 
-   6. gNBSim Deployment Features
+   gNBSim Deployment Features
 
     - Support deployment of gNBSim as standalone container
+
+# Support and Contributions
+
+The gnbsimF project welcomes new contributors. Feel free to propose a new feature or fix bugs!
```
[the remark this answers is not in the document]
Comment (thakurajayL) [reply README.md:140]: Done.

File: docs/gnbsim_apis.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+SPDX-License-Identifier: Apache-2.0
+
+-->
+
+## gNBsim can Optionally launch profiles through HTTP APIs
+
+    gNBSim can process HTTP Requests to launch profiles. For example running the
+    below curl command will launch a profile in gNBSim
+   
+    $ curl -i -X POST 127.0.0.1:6000/gnbsim/v1/executeProfile -H 'Content-Type: application/json' -d '{"profileType":"nwreqpdusessrelease","profileName":"profile8","enable":true,"gnbName":"gnb1","startImsi":"208930100007497","ueCount":1,"opc":"981d464c7c52eb6e5036234984ad0bcf","key":"5122250214c33e723a5dd523fc145fc0","sequenceNumber":"16f3b3f70fc2","defaultAs":"192.168.250.1","plmnId":{"mcc":"208","mnc":"93"}}'
```
[the remark this answers is not in the document]
Comment (thakurajayL) [reply docs/gnbsim_apis.md]: Fixed.

File: docs/config.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    List of gNB's to be simulated. Each item in the list holds 
+    configuration specific to a gNB.
+
+- profiles:
+    List of test/simulation profiles. Each item in the list holds 
+    configuration specific to a profile. Each profile executes   
+    required set of systems procedures for the configured set of 
+    IMSI's 
+        
+## Enable or disable a specific profile using the "enable" field.
```
Comment (gab-arrobo) [inline docs/config.md]:
```suggestion
Enable or disable a specific profile using the "enable" field
```

File: docs/config.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+<!--
+SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+SPDX-License-Identifier: Apache-2.0
+
+-->
+
+# Configure gNBSim
+    
+## The config file for gNBSim can be found [here](../config/gnbsim.yaml)
```
Comment (gab-arrobo) [inline docs/config.md:9]:
```suggestion
The config file for gNBSim can be found [here](../config/gnbsim.yaml)
```

File: docs/config.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+- profiles:
+    List of test/simulation profiles. Each item in the list holds configuration specific to a profile.
+    Each profile executes required set of systems procedures for the configured set of IMSI's.
+    Each profile has enable field and if set to true then that profile is executed.
+        
+## Description of Each Profile
+        
+Currently following profiles are supported :
+**1.register:**
```
Comment (gab-arrobo) [inline docs/config.md:25]:
```suggestion
Currently following profiles are supported :

**1.register:**
```

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**